mllib/src/main/scala/org/apache/spark/ml/feature/ChiSqSelector.scala

```diff
@@ -49,15 +49,13 @@ private[feature] trait ChiSqSelectorParams extends Params
    *
    * @group param
    */
-  @Since("1.6.0")
```

Comment:
Usually we don't add ```since``` tag to variables and functions in traits, since they may be inherited by new child classes later on and the tag is incorrect for them.